renderer/containers/Root.js

```diff
@@ -53,11 +67,31 @@ const Root = ({
         setMounted(true)
         await initDatabase()
         await initSettings()
-        await initTheme()
+        initTheme()
```

Comment:
maybe it's worth to introduce a composite init action outside of the `Root` component to avoid growing prop list ?